node.js/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,71 @@
+const { query } = require("express");
+const express = require("express");
+const app = express();
+let  reducer = (accumulator, currentValue) => accumulator + currentValue;
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/add", (req, res) => {
+    const QueryStringArr= Object.values(req.query)
+    const QueryIntParamArr = QueryStringArr.map(q => parseInt(q))
+    const sum = QueryIntParamArr.reduce(reducer)
+    res.json({sum});
```

Comment:
```suggestion
    const QueryStringArr = Object.values(req.query).flat();
  const sum = QueryStringArr.reduce(add);
  res.send(`${sum}`);
```
and similar for all operations